cucumber/authenticators_jwt/features/authn_jwt_security.feature

```diff
@@ -77,10 +77,10 @@ Feature: JWT Authenticator - Security
     CONJ00035E Failed to decode token (3rdPartyError ='#<JWT::IncorrectAlgorithm: Expected a different algorithm>')>
     """
 
-  Scenario Outline: ONYX-8858: Token Signed by RSA, 200 OK
-    Given I initialize JWKS endpoint with file "myJWKs.json"
-    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
-    And I issue an "<alg>" algorithm RSA JWT token:
+  Scenario Outline: ONYX-8858: Algorithms sanity
+    Given I initialize remote JWKS endpoint with file "ONYX-8858-<alg>" and alg "<alg>"
```

Comment:
you have specific file for each alg?

Reply:
Remote JWKS endpoint backend is stateful key value set, so each alg has to be under different key (file name).